src/Yesod/Markdown.hs

```diff
@@ -36,6 +37,10 @@ module Yesod.Markdown
 import Data.Monoid (Monoid)
 #endif
 
+#if MIN_VERSION_pandoc(2,0,0)
```

Comment:
I don't know that this package is important/popular enough to warrant such attention to backwards compatibility. I'd rather the code be simpler, cut a new major version, and just say `yesod-markdown >= x` for `pandoc-2`, `yesod-markdown < x` for `pandoc-1`.

Reply:
That definitely sounds simpler.  Let's do that.

Reply:
I've changed the version bounds to to be `pandoc >= 2.0 && < 2.1`.